src/main/resources/templates/typescript/exercise/src/sortstrategy.ts

```
@@ -0,0 +1,3 @@
+export default interface SortStrategy {
+    // TODO: Create a SortStrategy interface according to the UML class diagram
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)